test/parallel/test-dgram-send-address-types.js

```diff
@@ -5,27 +5,37 @@ const dgram = require('dgram');
 
 const buf = Buffer.from('test');
 
+const defaultCases = ['', null, undefined];
+
 const onMessage = common.mustSucceed((bytes) => {
   assert.strictEqual(bytes, buf.length);
-}, 6);
+}, defaultCases.length + 1);
 
 const client = dgram.createSocket('udp4').bind(0, () => {
   const port = client.address().port;
 
   // Check valid addresses
-  [false, '', null, 0, undefined].forEach((address) => {
+  defaultCases.forEach((address) => {
     client.send(buf, port, address, onMessage);
   });
 
   // Valid address: not provided
   client.send(buf, port, onMessage);
 
   // Check invalid addresses
-  [[], 1, true].forEach((invalidInput) => {
+  [
+    [],
+    1,
```

Comment:
Can we add 0 as it's a falsy value please?
```suggestion
    0,
    1,
```